air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -104,6 +146,11 @@ proc.evaluate_integrity_constraints
     ext2div # divide the numerator by the divisor
 end # END PROC evaluate_integrity_constraints
 
+# Procedure to compute the boundary constraint.
```

Comment:
minor nit: should this be "Procedure to evaluate all boundary constraints".

Reply:
changed